PhotoStream/app/src/main/java/com/example/siyer/photostream/CollDbHelper.java

```diff
@@ -0,0 +1,87 @@
+package com.example.siyer.photostream;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+/**
+ * Created by siyer on 9/30/2015.
+ * Database methods to create/add/delete from the SQL dtatbase
+ */
+import com.example.siyer.photostream.CollectionSchema.CollTable;
+
+import java.util.ArrayList;
+
+public class CollDbHelper extends SQLiteOpenHelper {
+
+    public static final int DATABASE_VERSION = 5;
+    public static final String DATABASE_NAME = "FeedReader.db";
+
+    public CollDbHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        db.execSQL(CollTable.SQL_CREATE_ENTRIES);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        db.execSQL(CollTable.SQL_DELETE_ENTRIES);
+        onCreate(db);
+    }
+
+    @Override
+    public void onDowngrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        onUpgrade(db, oldVersion, newVersion);
+    }
+
+    public void writeDatabase(String input){
+        SQLiteDatabase writeDBase = getWritableDatabase();
```

Comment:
You can move this call to your oncreate for performance increase. Define two sql databases as private params and then instantiate them in your oncreate.